src/sacn/common.c

```diff
@@ -101,6 +105,8 @@ etcpal_error_t sacn_init(const EtcPalLogParams* log_params)
       source_initted = ((res = sacn_source_init()) == kEtcPalErrOk);
     if ( res == kEtcPalErrOk)
       merger_initted = ((res = sacn_dmx_merger_init()) == kEtcPalErrOk);
+    if (res == kEtcPalErrOk)
+      merge_receiver_initted = ((res = sacn_merge_receiver_init()) == kEtcPalErrOk);
```

Comment:
Let's refactor this to look more like https://github.com/ETCLabs/RDMnet/blob/develop/src/rdmnet/core/common.c#L141 at some point.